src/firecracker/tests/test_api_server.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const BINARY: &str = env!("CARGO_BIN_EXE_firecracker");
```

Comment:
Please don't make integration tests in Rust. We already have the Python integration tests for that. 

Furthermore, this test seems to duplicate https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/functional/test_api_server.py#L10

Reply:
I don't think the existing tests look at stdout and stderr. They only check for the presence of the error in the log target (and check the exit-code is non-zero but not a specific value).

Reply:
> We have existing integration tests in https://github.com/firecracker-microvm/firecracker/blob/main/src/vmm/tests/integration_tests.rs. I found this the simplest test to specifically cover the change. I don't see a reason for not having this additional test?

To me that's still a unit test, as it does not use the firecracker binary. That's where I make the distinction.